### DISCUSSION TITLE
How to connect chatwoot repo to the my heroku app

### DISCUSSION THREAD

**Discussion by NiravSa** [chatwoot/chatwoot#4844] [Self hosted]
Whenever a new version is out for chatwoot, you update your Heroku deployment through the following steps.

1. In the deploy tab, choose Github as the deployment option.
2. Connect chatwoot repo to the app.
3. Head over to the manual deploy option, choose a master branch and hit deploy.

I am stuck with the second step. don't know how to connect chatwoot repo to my Heroku app. when I click on Connect to GitHub button it opens the GitHub auth popup and connects my GitHub account to Heroku app.

Thanks

**Comment by sojan-official** [comment]
@NiravSa You can fork the chatwoot repo to your profile. So that it will be available to connect under your account. You can update the fork with upstream changes whenever chatwoot releases a new update
